workflow-templates/python-cross-platform-support.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           python-version: 3.11
 
       - name: Install ${{ github.event.repository.name }} Package
-        run: python -m pip install -e .[all]
+        run: pip install -e .[all]
```

Comment:
Why is this changed?  And should it also be changed in the core aac repo?

Reply:
From testing I did in my forked repo, it wasn't necessary and this is the only spot that our pip install had the preface of "python -m". So, removed it to be consistent, and not worried about updating it in core with this effort because we have a task for pulling all of the new template workflows here shortly. But till we are there, we need main to be working correctly so just applied the doc deployment change.